index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script defer src="script.js"></script>
+    <title>Document</title>
+</head>
+<body>
+
+    <div class= "movie-title">
+        <h1>🍿🎥FLIXSTER</h1>
+    </div>
+    <form id="search-form">
+        <input id="search-input" required name="search" placeholder="Enter movie name..." class="" />
+        <button id="srch-button" type="submit" class="search-btn">Search now playing!</button>
```

Comment:
be consistent with id and class names; e.g. `id="search-button"` since all other references to search have it spelled out in full (vs. `id="srch-button"`)